VideoLocker/src/main/java/org/edx/mobile/player/VideoListFragment.java

```diff
@@ -441,12 +440,8 @@ public void onSelectItem() {
             videoListView.setOnItemClickListener(adapter);
 
             setActivityTitle(enrollment.getCourse().getName());
-            try{
-                environment.getSegment().screenViewsTracking("My Videos - All Videos - "
-                        + enrollment.getCourse().getName());
-            }catch(Exception e){
-                logger.error(e);
-            }
+            environment.getSegment().trackScreenView("My Videos - All Videos - "
+                    + enrollment.getCourse().getName());
```

Comment:
@aleffert: Is there a list of screen names anywhere? Is this really the right screen name?